pkg/controller/issueapi/issue.go

```diff
@@ -296,3 +305,26 @@ func (c *Controller) getAuthorizationFromContext(r *http.Request) (*database.Aut
 
 	return authorizedApp, user, nil
 }
+
+func (c *Controller) recordCapacity(ctx context.Context, realm *database.Realm) error {
+	if !realm.AbusePreventionEnabled {
+		return nil
+	}
+
+	dayStart := time.Now().UTC().Truncate(24 * time.Hour)
+	dayEnd := dayStart.Add(1)
+
+	realmStats, err := realm.Stats(c.db, dayStart, dayEnd)
+	if err != nil {
+		return err
+	}
+	if len(realmStats) != 1 {
+		return errors.New("found more than one row of daily stats")
+	}
+
+	dailyStats := realmStats[0]
+	capacity := 1 - (float64(dailyStats.CodesIssued) / float64(realm.AbusePreventionEffectiveLimit()))
```

Comment:
I'm not sure I understand why this is 1 - %?

Reply:
It felt wrong to alert above a capacity of 90% but maybe the wording on the alert should be "capacity utilization" so changed to that.